lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
```

Comment:
Everything works with an earlier bundler as well, we just won't get the speed benefit of this branch, which feels like an ok tradeoff.